projects/addon-doc/components/code/code.style.less

```diff
@@ -26,12 +21,22 @@
 
 .t-code-actions {
     position: absolute;
-    top: 0.875rem;
-    right: 2rem;
+    top: 0.75rem;
+    right: 0.75rem;
     display: flex;
-    opacity: 0;
-    gap: 0.5rem;
-    flex-direction: row-reverse;
+    gap: 0.375rem;
+    justify-content: center;
+    align-items: center;
+}
+
+.t-copy {
+    opacity: 0.5;
+    cursor: pointer;
+    transition: opacity 0.3s ease;
+
+    &:hover {
```

Comment:
Can it be a tuiIconButton appearance="icon" instead of manually controlling opacity? Would be keyboard focusable as well.

Reply:
fixed